rclcpp/include/rclcpp/topic_statistics/subscription_topic_statistics.hpp

```diff
@@ -152,6 +152,12 @@ class SubscriptionTopicStatistics
       publisher_->publish(msg);
     }
     window_start_ = window_end;
+    {
```

Comment:
Could you please add a unit test as well? Could be simple to check if a max or min is now outside of a window (see the existing test fixtures [here](https://github.com/ros2/rclcpp/blob/master/rclcpp/test/rclcpp/topic_statistics/test_subscription_topic_statistics.cpp)).

Reply:
OK.
I'll try to add a unit test as well.

Reply:
I've added a unit test.
Would you please check if it is the test you were expecting?